bot.py

```diff
@@ -157,9 +184,27 @@ async def get_last_match(ctx, name: str):
         embed.set_image(url="attachment://df_styled.png")
         await ctx.send(embed=embed, file=file)
         os.remove("df_styled.png")
-        # pylint: disable=broad-except
+
+    # pylint: disable=broad-except
     except Exception as e_values:
-        print(e_values)
+        # 404 error means Data not found in API
+        if "404" in str(e_values):
+            error_title = f'Summoner "{name}" is not found'
+            error_description = f"Please check the summoner name agian \n \
+              \n __*NOTE*__ :   **{get_last_match.name}** command only accepts one summoner name.\
+              \n\n Please type  `last_match --help`  to see how to use"
+
+        else:
+            error_title = "Error"
+            error_description = "Oops! Something went wrong.\n\nPlease try again!\
+              \n\n Please type  `last_match --help`  to see how to use"
```

Comment:
What does this look like on discord? `error_description` seems pretty long..
This can be shortened into 
```
error_description = "Oops! Something went wrong.\n\n Please type `last_match --help` to see how to use and try again!"
```

Reply:
The image shown in discord is attached above.

The reason why I use `Please try again` is that I considered situations where the user input correctly but there could be a system error. However, if you think `Please try agian` doen not need, I will take it out.

Also, `\n\n` which follows `\`(line #199) needs to fix the mobile version of indentation. 
if I use only one `\n` the embed will be shown like
<img width="401" alt="Screen Shot 2021-07-13 at 3 44 13 AM" src="https://user-images.githubusercontent.com/47313362/125412112-9b939880-e38c-11eb-8c2f-6418c0b4d91b.png">

Lastly, `\n\n` which follows `something went wrong` is my mistake. Only one `\n` needs in there.

This explanation will be also the answer to the two reviews below.
https://github.com/dhmoon91/discord/pull/27#discussion_r668374187
https://github.com/dhmoon91/discord/pull/27#discussion_r668374501